Nighttime-Operations/Auxiliary-Telescope/AT-Pre/Prepare-for-On-Sky-Operations.rst

```diff
@@ -0,0 +1,118 @@
+
+#############################
+Prepare for On-Sky Operations
+#############################
+
+.. _Open-for-On-Sky-Operations-Overview:
+
+Overview
+========
+
+This procedure gets the AuxTel system ready for on-sky operations, preparing the telescope and dome to be fully open. 
+
+In a standard night, it will follow the execution of the :ref:`venting <AT-Pre-Prepare-for-vent>` procedure, but it works from any initial state of the system. 
+
+If venting wasn't possible, observers must prepare for on-sky once the Sun is below an elevation of 25 degrees. 
+The Sun's coordinates at any given time, along with several other useful ephemeris, can be found at the :ref:`sky almanac <Sky-Almanac-Get-Sun-Coordinates>`.
+
+.. _Open-for-On-Sky-Operations-Prerequisites:
+
+Prerequisites
+=============
+
+* Before making the decision to open, review the weather conditions and :ref:`weather constraints <weather-constraints-deciding-to-open>` page.
+
+* AuxTel is fully ready to operate and all components are enabled. 
+
+* The :ref:`daytime checkout <AT-Daytime-Checkout-Daytime-Checkout>` has been executed successfully. 
+
+* This procedure will take the system from venting, the :ref:`calibration <LATISS-Combined-Calibrations-Procedure-LATISS-Combined-Calibrations-Generation-Procedure>` 
+  state or any other initial state to be prepared for on-sky observations. 
+
+.. _Open-for-On-Sky-Operations-Post-Condition:
+
+Post-Condition
+==============
+
+- Telescope is ready for on-sky operations. 
+  
+  Around 15 minutes before the end of the evening nautical twilight, proceed with the :ref:`beginning of the night procedure to center, align and focus the telescope. <AT-On-sky-WEP>`
+
+.. _Open-for-On-Sky-Operations-Procedure-Steps:
+
+Procedure Steps
+===============
+#. Review :ref:`prerequisites <Open-for-On-Sky-Operations-Prerequisites>` and confirm they have been executed. 
+
+#. Announce in the #summit-announce and #summit-auxtel Slack channel that AuxTel is preparing to go on-sky. 
+
+#. Command the telescope and dome to prepare for on-sky observations. 
+
+   Load the ``auxtel/prepare_for/onsky.py`` script from the LOVE ``ATQueue`` panel, under ``AVAILABLE SCRIPTS`` by clicking on the blue icon. 
+   The script will be added to the script queue. 
+   
+   .. note::
+     If the ``auxtel/prepare_for/vent.py`` script is still running, the queue will run ``auxtel/prepare_for/onsky.py`` once venting is done, around 20 minutes before sunset.  
+
+   In the ``Configuring script:onsky`` screen that pops up, leave the ``CONFIG`` box empty. 
+   Make sure there are no spaces in the ``CONFIG`` box, as it may prevent the proper configuration and loading of the script. 
+
+   .. figure:: ./_static/PrepareforOnSky_AuxTel.png
+     :name: prepareforonsky_AuxTel
+    
+     Screenshot of LOVE launching ``auxtel/prepare_for/onsky.py`` script. 
+
+   The script includes the following steps:
+
+       * Slew telescope to park position. 
```

Comment:
It's not actually the park position it slews it to a position where it is the least likely to be hit by the sun.